--- a/cuda/private/templates/BUILD.cublas
+++ b/cuda/private/templates/BUILD.cublas
@@ -38,5 +38,36 @@
     ]) + if_windows([
         ":cublasLt_lib",
         ":cublas_lib",
     ]),
 )
+
+cc_import(
+    name = "cublas_static_a",
+    static_library = "%{component_name}/%{libpath}/libcublas_static.a",
+    target_compatible_with = ["@platforms//os:linux"],
+)
+
+cc_import(
+    name = "cublasLt_static_a",
+    static_library = "%{component_name}/%{libpath}/libcublasLt_static.a",
+    target_compatible_with = ["@platforms//os:linux"],
+)
+
+cc_library(
+    name = "cublas_static",
+    deps = [
+        ":%{component_name}_headers",
+    ] + if_linux([
+        ":cublasLt_static_a",
+        ":cublas_static_a",
+    ]),
+)
+
+cc_library(
+    name = "cublasLt_static",
+    deps = [
+        ":%{component_name}_headers",
+    ] + if_linux([
+        ":cublasLt_static_a",
+    ]),
+)
